src/viewer/reference/native_plugins.md

```diff
@@ -198,6 +198,56 @@ const viewer2dPlugin = this.$viewer.localContext.getPlugin("viewer2d");
 | `syncRotationFrom3DCamera(eye: [number,number,number], look: [number,number,number],up: [number,number,number]): void` | **Default** arguments are `eye = [0, 0, 0], look = [0, 0, 0],up = [0, 1, 0]`. Synchronize the roation between the given 3D camera parameters and the 2D camera. |
 
+## Viewer DWG
+
+- name: dwg
+
+This plugin display the DWG representation of the IFC.
+
+### Events
+
+| Name                 | Payload           | Description                           | Emitted on                         |
+| :------------------- | :---------------- | :------------------------------------ | :--------------------------------- |
+| `dwg-model-loaded`   | `{ dwg, plugin }` | Emitted when a DWG model is loaded.   | `localContext` and `globalContext` |
+| `dwg-model-unloaded` | `{ dwg, plugin }` | Emitted when a DWG model is unloaded. | `localContext` and `globalContext` |
+
+### Instance API
+
+This API is available from this object on the `dwg` window:
+
+```javascript
+const viewerDWGPlugin = this.$viewer.localContext.getPlugin("dwg");
+```
+
+| Name                                                                                                                   | Description                                                                                                                                                     |
+| :--------------------------------------------------------------------------------------------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `selectOnClick: boolean`                                                                                               | **Default** to `true`. If true, clicking an object select it.                                                                                                   |
+| `highlightOnHover: boolean`                                                                                            | **Default** to `true`. If true, hovering an object highlight it.                                                                                                |
+| `viewer: E2D.Viewer`                                                                                                   | The [engine 2D viewer](https://2d-engine.bimdata.io).                                                                                                           |
+| `hideAll(): void`                                                                                                      | Hide all objects.                                                                         |
+
+## Viewer DXF
+
+Same api as the [DWG viewer](#viewer-dwg).
```

Comment:
The event are also `dwg-model-loaded` for DXF ?

Reply:
Not after https://github.com/bimdata/viewer/pull/783 merged :P